ray-operator/controllers/ray/utils/constant.go

```diff
@@ -154,3 +157,11 @@ const (
 	HeadService    ServiceType = "headService"
 	ServingService ServiceType = "serveService"
 )
+
+func RayOriginatedFromServiceLabelValue(name string) string {
```

Comment:
Can you add comments for these two functions? This function should be the only place to associate the RayService CR and its resources. We should define it clearly.

Reply:
Added: `This is also the only function to construct label filter of resources originated from a given RayService.`